package.json

```
@@ -27,15 +27,28 @@
 		"lint:fix": "eslint --fix .",
 		"format":
 			"prettier --write \"./*.{js,json,md}\" \"{api,config,docs,helpers,logic,modules,schema,scripts,storage,tasks,test,framework}{,/**}/*.{js,json,md}\"",
-		"test": "grunt",
-		"grunt": "grunt",
+		"test": "node framework/test/mocha/common/lisk-mocha-runner/index.js",
 		"mocha": "mocha",
 		"jest:unit":
 			"jest --config=./framework/test/jest/config/unit/jest.config.js",
 		"jest:integration":
 			"jest --config=./framework/test/jest/config/integration/jest.config.js",
 		"jest:functional":
 			"jest --config=./framework/test/jest/config/functional/jest.config.js",
+		"mocha:unit":
+			"node framework/test/mocha/common/lisk-mocha-runner/index.js unit",
```

```suggestion
			"node framework/test/mocha/common/lisk-mocha-runner unit",
```
since it's `index.js` i think you don't need to specify file